recipe/meta.yaml

```diff
@@ -675,7 +696,7 @@ outputs:
         - onednn_third-party-programs.txt
         - runtime-third-party-programs.txt
     build:
-      string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+      noarch: python
```

Comment:
could you please remove all the things except this line? 
Dependencies are handled here https://github.com/conda-forge/openvino-feedstock/pull/71

Reply:
i'll let you followthough with #71